src/main/java/uk/gov/hmcts/reform/blobrouter/data/events/EventType.java

```diff
@@ -9,4 +9,6 @@
     DUPLICATE_REJECTED,
     ERROR,
     NOTIFICATION_SENT,
+    MANUALLY_MARKED_AS_REJECTED,
+    MANUALLY_MARKED_AS_DISPATCHED
```

Comment:
do we use event type in any report or would it affect any reports?

Reply:
In this report
![image](https://user-images.githubusercontent.com/2663436/108532712-06eea800-72d0-11eb-9d3c-717a3b111fdb.png)

Reply:
I mean if we are classifying with event type that would affect the report result.